TWLight/templates/home.html

```diff
@@ -68,7 +68,7 @@ <h2 style="padding-bottom:20px;">{% trans "Library Bundle" %}
             </li>
           {% endfor %}
         </ul>
-        {% if user.is_authenticated and not user.editor.wp_not_blocked %}
+        {% if user.is_authenticated and not user.editor.ignore_wp_blocks and not user.editor.wp_not_blocked %}
```

Comment:
As far as I understand, the `ignore_wp_blocks` flag should be on to ignore the blocks, so this condition has to be: 

```suggestion
        {% if user.is_authenticated and user.editor.ignore_wp_blocks and not user.editor.wp_not_blocked %}
```

@Samwalton9 is this correct?

Reply:
This line is a little confusing because the block field is `not_blocked` so we're looking at `not not_blocked` (blocked) to see if we should show this message. It decides whether we show the 'you are blocked message' and we only want to do that if we _haven't_ ignored their WP blocks.